specs/splitbutton/splitbutton.spec.md

```diff
@@ -0,0 +1,220 @@
+# Split Button
+
+This is the working branch for the `split-button` component spec. It is still in development and implementation details are rough but should be ready for high-level feedback and discussion.
+
+## Overview
+
+A `fast-split-button` is a primary action button and a collection of related action buttons. The primary button is shown and the related buttons are hidden in a menu. It takes takes two forms:
+
+ 1. **Stateless split-button**: a primary action button paired with a menu-button which shows or hides a dropdown collection of related buttons, as seen here in Teams:
+
+![image of a split button from Microsoft Teams](images/split-button_example_Teams.png)
+
+ 2. **Stateful split button**: a button and popover as seen here in Facebook's "Like" button:
```

Comment:
What do we technically mean by the component being "stateful".

Reply:
A stateful split button is one that stores state, similar to a "like" button. The opposite would be a similar to the email "Send | Send Later" button.